xrpl_types/src/lib.rs

```diff
@@ -1,16 +1,18 @@
 //! Core types and related functions for the XRP Ledger. Reused between Web and
 //! WebSocket clients and potentially for server-side code.
 
-pub mod account;
-pub mod amount;
-pub mod book;
-pub mod currency;
-pub mod ledger_index;
-pub mod transaction;
+mod account;
```

Comment:
I think the convention is not to have public modules if you "`pub use`" their contents. Else the types will be represented at multiple paths.

Reply:
I wonder if it would be better to remove the `pub use` instead?